addons/Fractions/src/presenter.js

```diff
@@ -1145,15 +1173,32 @@ function AddonFractions_create(){
         presenter.eventBus.sendEvent('ValueChanged', eventData);
     };
 
+    presenter.onEventReceived = function (eventName, data) {
+        switch (eventName) {
+            case "ShowAnswers":
+                presenter.showAnswers();
+                break;
+            case "HideAnswers":

```

Comment:
skoro hideAnswers i gradualHideAnswers działają w obrębie jednej funkcji to można zapisać:
case "HideAnswers":
case "GradualHideAnswers":
_ presenter.hideAnswers();
_ break;

Reply:
Celowo rozdzieliłem funkcjonalność HideAnswers i GradualHideAnswers. W ten sposób łatwiej będzie wprowadzić ewentualne poprawki lub rozwinięcia.